pkg/component/runtime/service.go

```diff
@@ -38,6 +39,9 @@ var (
 	ErrOperationSpecUndefined = errors.New("operation spec undefined")
 	// ErrInvalidServiceSpec error invalid service specification.
 	ErrInvalidServiceSpec = errors.New("invalid service spec")
+
+	serviceRestartDelay   = 30 * time.Second // variable so it can be shortened in tests
```

Comment:
instead of setting this as a variable in the whole module could this be moved to be a variable in the struct and then set this value in the struct in the tests?

Reply:
Looks good.